src/libraries/Microsoft.PowerFx.Core/Public/Types/RecordType.cs

```diff
@@ -96,30 +96,5 @@ public static RecordType Empty()
         {
             return new KnownRecordType();
         }
-
-        /// <summary>
-        /// Resolve logical names when display names are given.
-        /// </summary>
-        /// <param name="recordToResolve">RecordValue to resolve.</param>
-        /// <returns>RecordValue containing only logical field names.</returns>
-        internal RecordValue ResolveToLogicalNames(RecordValue recordToResolve)
```

Comment:
We can remove this because the binder can actually handle it and have the logical names in the IR (which is even better than having to resolve them at runtime).